src/components/Tabs/Tabs.scss

```diff
@@ -327,24 +326,88 @@ $focus-state-box-shadow-color: rgba(0, 0, 0, 0.8);
 
   .Tab {
     padding: spacing(tight) spacing(extra-tight);
-
-    &:active .Title::before {
-      background: var(--p-action-primary-pressed);
+    &:hover .Title {
+      @media (-ms-high-contrast: active) {
+        outline: rem(1px) solid ms-high-contrast-color('text');
+        &::before {
+          display: none;
+        }
+      }
     }
 
     &:focus .Title::before {
       background: var(--p-border-hovered);
     }
+
+    &:active .Title::before {
+      background: var(--p-surface-primary-selected-pressed);
+    }
   }
 
   .DisclosureActivator {
-    margin: 1px 0 -1px;
-    padding: spacing(tight) spacing(tight) spacing(tight) 0;
+    &:hover .Title {
+      background: none;
+
+      &::before {
+        background: var(--p-border-hovered);
+      }
+
+      @media (-ms-high-contrast: active) {
+        outline: rem(1px) solid ms-high-contrast-color('text');
+
+        // stylelint-disable-next-line max-nesting-depth
+        &::before {
+          display: none;
+        }
+      }
+    }
+
+    &:focus .Title {
+      @include focus-ring($style: 'focused');
+
+      &::before {
+        background: var(--p-border-hovered);
+      }
+      @media (-ms-high-contrast: active) {
+        outline: rem(1px) solid ms-high-contrast-color('text');
+
+        // stylelint-disable-next-line max-nesting-depth
+        &::before {
+          display: none;
+        }
+      }
+    }
+
+    &:active .Title::before {
+      background: var(--p-surface-primary-selected-pressed);
+    }
   }
 
   .Tab-selected {
-    &:focus .Title::before {
-      background: var(--p-action-primary);
+    &:hover .Title {
+      @media (-ms-high-contrast: active) {
+        outline: rem(3px) solid ms-high-contrast-color('text');
+
+        // stylelint-disable-next-line max-nesting-depth
+        &::before {
+          display: none;
+        }
+      }
+    }
+
+    &:focus .Title {
+      @media (-ms-high-contrast: active) {
```

Comment:
Since I am seeing this same code more or less repeated, it might make sense to abstract this into a `mixin`:

```scss
@mixin focus-ring-ms-high-contrast($border-width: border-width()) {
  @media (-ms-high-contrast: active) {
    outline: $border-width solid ms-high-contrast-color('text');

    &::before {
      content: none;
    }
  }
}

.Something {
  &:hover .Title {
    @include focus-ring-ms-high-contrast($border-width: border-width(thicker));
  }
}
```

Thoughts?

Reply:
OOH LA LA, this is awesome. YEs. I think this is MUCH cleaner. I'll implement this.